experimental/LieAlgebras/src/PathModel.jl

```diff
@@ -0,0 +1,617 @@
+##### Kashiwara Crystal #####
+
+abstract type AbstractCrystal <: AbstractAlgebra.Set end
```

Comment:
Please add a docstring to all new types that briefly explains what it is about (this could also contain a citation of the reference work I mention in another comment)